packages/http-server/test/integration/http-server.integration.ts

```diff
@@ -0,0 +1,112 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/http-server
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+import {HttpServer} from '../../';
+import {supertest, expect} from '@loopback/testlab';
+import * as makeRequest from 'request-promise-native';
+
+describe('HttpServer (integration)', () => {
+  it('starts server', async () => {
+    const server = new HttpServer(
+      (req, res) => {
+        res.end();
+      },
```

Comment:
Please extract this request handler that's repeated in all tests into a shared helper function.

```ts
const server = new HttpServer(
  dummyRequestHandler,
  // ...
);
```